src/main/java/wellnus/atomichabit/command/AddCommand.java

```diff
@@ -78,7 +95,7 @@ protected String getFeatureKeyword() {
      * After that, print a message telling the user what the new habit added is
```

Comment:
Update JavaDoc to include `@throws` description on the new `AtomicHabitException`

Reply:
Resolved in 38c723b.